### PR TITLE
[SD-3424] - Improvements in associated items

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
                     <div class="basic item-block" 
                         ng-click="view(item)">
                         <div class="content">
-                            <p><i class="filetype-icon-{{item.type}}"></i><span class="keyword" ng-if="item.slugline">{{ item.slugline }}</span> {{item.headline || item.type}}</p>
+                            <p><i class="filetype-icon-{{item.type}}"></i><span class="keyword" ng-if="item.slugline">{{ item.slugline }}</span> {{item.headline || item.type}} {{item.anpa_take_key}}</p>
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html]: It would be good if we display the `item.type` always.

Comment (akintolga) [inline client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html]: Item type is displayed as the icon

Comment (sivakuna-aap) [inline client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html]: Right. What I meant was the icon should be displayed always and shouldn't be ignored if headline exists.

**Comment by marwoodandrew** [comment]
:+1:

**Pull request merged**